src/base/projects/Issue.svelte

```diff
@@ -136,28 +133,20 @@
   </header>
   <main>
     <div class="comments">
-      <Comment comment={issue.comment} {getImage} {wallet} />
       {#each issue.discussion as comment}
         <Comment {comment} {getImage} {wallet} />
-        {#if comment.replies}
-          <div class="replies">
-            {#each comment.replies as reply}
-              <Comment comment={reply} {getImage} {wallet} />
-            {/each}
-          </div>
-        {/if}
       {/each}
     </div>
     <div class="metadata layout-desktop">
       <div class="metadata-section">
-        <div class="metadata-section-header">Labels</div>
+        <div class="metadata-section-header">Tags</div>
         <div class="metadata-section-body">
-          {#if issue.labels?.length}
-            {#each issue.labels as label}
-              <span class="label">{label}</span>
+          {#if issue.tags?.length}
+            {#each issue.tags as tag}
+              <span class="label">{tag}</span>
             {/each}
           {:else}
-            <div class="metadata-section-empty">No labels.</div>
+            <div class="metadata-section-empty">No tags.</div>
```

Comment:
I don't have much context on the heartwood implementation, could you elaborate why we rename here?

Reply:
This was originally renamed by mistake and then I just went along with it, so now we're just making sure it's consistent everywhere :D